model/tasks/CompileDelivery.php

```diff
@@ -131,6 +131,6 @@ public static function createTask(\core_kernel_classes_Resource $test, \core_ker
             $parameters['deliveryClass'] = $deliveryClass->getUri();
         }
 
-        return $queueDispatcher->createTask($action, $parameters, __('Publishing of "%s"', $test->getLabel()), null, true);
+        return $queueDispatcher->createTask($action, $parameters, __('Publishing of <b>%s</b>', $test->getLabel()), null, true);
```

Comment:
Just to be on the safe side, we should html-encode the label value here, since the front end stopped doing it.
Otherwise it may be prone to stored XSS attacks.

```suggestion
        return $queueDispatcher->createTask(
            $action,
            $parameters,
            __('Publishing of <b>%s</b>', htmlspecialchars($test->getLabel())),
            null,
            true
        );
```